Problem1.c

```diff
@@ -0,0 +1,102 @@
+/* Problem Statement:
+Verified on leetcode
+
+In Pascal's triangle, each number is the sum of the two numbers directly above it.
+
+Example:
+
+Input: 5
+Output:
+[
+     [1],
+    [1,1],
+   [1,2,1],
+  [1,3,3,1],
+ [1,4,6,4,1]
+]
+
+
+ *
+ * Time Complexity : O(n^2) worst case
```

Comment:
why did you mention the worst case
isnt this for every case equal to n*(n+1)/2 which is equal to O(N)

Reply:
Hi @kaurramanpreet , Yes, it's wrong it should be O(N). Here I didn;t take into account that only the last iteration of the loop will be running for N entries. it's a progressive one. I will fix it.
Thanks for pointing it out